front-end/src/AdminBar/adminStyles.js

```diff
@@ -13,8 +14,10 @@ export const AdminPanel = styled.div`
     grid-template-areas:
      "clear undo send"
      "currentInput currentInput currentInput";
+    width: 100%; 
+    padding: 0%;
```

Comment:
The nit of all nits: It's quite common to prefer unitless values for `0`, but I also hate myself a tiny bit for commenting on it

Reply:
I do prefer them as well! No worries 💯